builtin/providers/google/resource_compute_instance_group_test.go

```diff
@@ -201,6 +227,44 @@ func testAccComputeInstanceGroup_named_ports(n string, np map[string]int64, inst
 	}
 }
 
+func testAccComputeInstanceGroup_hasCorrectNetwork(n_ig string, n_nw string, instanceGroup *compute.InstanceGroup) resource.TestCheckFunc {
```

Comment:
style nit: I'm not a huge fan of mixing types of cases- let's leave everything camelCase